style-guide/styles/wds-variables/_buttons.scss

```diff
@@ -0,0 +1,3 @@
+$button-active-color: darken($color-link, 20%);
+$button-line-height: 1.3em;
+$hover-transition-duration: 300ms;
```

Comment:
Do we want to export it as DS variable? If so then probably we need to make the name more descriptive.